docs/developer/eventing/sinks/creating-event-sinks/writing-event-sinks-with-golang/README.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "Writing an event sink using Golang"
+weight: 10
+type: "docs"
+showlandingtoc: "false"
+---
+
+# Writing an event sink using Golang
+
+This tutorial provides instructions to build an event sink in Golang and implement it as a ksvc.
+
+## A Quick Overview of Building and Deploying a Simple Event Sink with Golang
+
+Below we will create a simple Sink that receives CloudEvents and performs an example operation on them.
+
+From within the `$GOPATH`, Create a folder `demosink` with a new file `main.go`:
```

Comment:
Make these numbered steps please so that it's easier for folks to follow

Reply:
Fixed: https://github.com/knative/docs/pull/4208/commits/6f43db24c1ef811a54d9de91acf0bcc1f2c9aa68 
Thank you!